src/components/CollectionDetailPage/CollectionDetailPage.container.ts

```diff
@@ -30,7 +30,7 @@ const mapState = (state: RootState): MapStateProps => {
 }
 
 const mapDispatch = (dispatch: MapDispatch): MapDispatchProps => ({
-  onNavigate: path => dispatch(push(path)),
+  onNavigate: (path, prop) => dispatch(push(path, prop)),
```

Comment:
What does prop stand for in this case? Can we rename it to something more representative?

Reply:
like routerProps?